setup/functions.py

```diff
@@ -665,12 +665,16 @@ def add_config(obj_or_filename, num_spaces=0, label=""):
     spaces = " " * num_spaces
     contents = ""
     indented_contents = ""
-    try:
-        with open(obj_or_filename, 'r') as f:
-            contents = f.read()
-    except FileNotFoundError:
-        # not a file
-        contents = obj_or_filename
+
+    contents = obj_or_filename
+
+    if len(obj_or_filename.split('\n')) == 1 :
+        try:
+            with open(obj_or_filename, 'r') as f:
+                contents = f.read()
+        except FileNotFoundError:
+            # not a file
+            contents = obj_or_filename
```

Comment:
Nit: can we just `pass` in the except block since it doesn't change the value of `contents`?